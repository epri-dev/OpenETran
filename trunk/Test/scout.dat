time 4 0.01e-6 15.0e-6

* overhead line geometry
span 1
conductor 1 10.0 -1.5 0.00715   3854.0
conductor 2 10.5  0.0 0.00715 -11097.0
conductor 3 10.0  1.5 0.00715   7243.0
conductor 4  8.0  0.0 0.00715      0.0

* single-phase underground attached to upper phase
span 2
cable 2 30.0 1.5e8 -11097.0
end
end

* overhead line spans
line 1 2 1 60.0 1 0
line 2 3 1 60.0 0 0
line 3 4 1 60.0 0 0
line 4 5 1 60.0 0 1
*underground spans
line 3 6 2 270.0 0 0
line 6 7 2  30.0 0 0
end

labelphase 0 G
labelphase 1 A
labelphase 2 B
labelphase 3 C
labelphase 4 N

labelpole 3 riser
labelpole 6 xfmr
labelpole 7 opntie

ground 85.0 250.0 400.0e3 0.5e-6 10.0
pairs 4 0
poles 1 2 3 4 5

* riser pole, scout, and open tie arresters
arrbez 0.0e3 40.0e3 0.0 0.0 0.0 0
pairs 2 0
poles 2 3 4 7

surge -10.0e3 1.0e-6 50.0e-6 0.0e-6
pairs 2 0
poles 1

meter
pairs 2 0
poles 3 6 7
