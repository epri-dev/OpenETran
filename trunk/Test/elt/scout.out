  N   span     dT   Tmax
  4   0.00 1e-008 1.5e-005
Z-phase
  4.76183e+002  1.53897e+002  1.14495e+002  1.18652e+002
  1.53897e+002  4.79110e+002  1.53897e+002  1.20089e+002
  1.14495e+002  1.53897e+002  4.76183e+002  1.18652e+002
  1.18652e+002  1.20089e+002  1.18652e+002  4.62794e+002
Z-modal
  3.09135e+002  2.27374e-013  -3.47283e-014  -4.97380e-014
  8.17124e-014  8.65241e+002  -1.49568e-014  0.00000e+000
  -4.83315e-015  -5.64278e-014  3.61688e+002  -1.86397e-015
  -7.10543e-015  0.00000e+000  4.04593e-014  3.58204e+002
Modal Transformation (Ti)
  4.62950e-001  4.99988e-001  -7.07107e-001  -1.88916e-001
  -7.43996e-001  5.40392e-001  2.31700e-015  -3.92998e-001
  4.62950e-001  4.99988e-001  7.07107e-001  -1.88916e-001
  -1.33509e-001  4.56071e-001  -6.75388e-015  8.79872e-001
Power-frequency source at pole 1.
   #        vp        vm         i
   1    3854.0   13393.5      13.9
   2  -11097.0    -448.4     -35.0
   3    7243.0    2396.4      23.3
   4       0.0    2264.7      -0.5
Power-frequency source at pole 5.
   #        vp        vm         i
   1    3854.0   13393.5      13.9
   2  -11097.0    -448.4     -35.0
   3    7243.0    2396.4      23.3
   4       0.0    2264.7      -0.5
Meter at pole 3, from 2 to 0, max V   = -3.233520e+004
Meter at pole 6, from 2 to 0, max V   = -3.195587e+004
Meter at pole 7, from 2 to 0, max V   = -3.156447e+004
nr_iter = 21477, nr_max = 9
Execution time [sec]: 0.046
